### PR TITLE
159 zoom in and zoom out buttons on the 1st step page of report hazard should be the same as we have on the homepage

### LINKED ISSUE #159: Zoom in and zoom out buttons on the 1st step page of Report Hazard should be the same as we have on the homepage
image

image

### PR DESCRIPTION
**Pull request by BryanCaldeira**
fix #159 

Zoom controls added to Report Hazard map, styling done as per the design in home screen zoom controls

### CODE REVIEW

File: pages/report-hazard/index.html [hunk 1]
```diff
@@ -17,6 +17,8 @@
   <link rel="stylesheet" href="../../modules/leaflet/leaflet.css" />
   <!-- leaflet JS for maps -->
   <script src="../../modules/leaflet/leaflet.js"></script>
+  <!-- Map Controls Styles -->
+  <link rel="stylesheet" href="/assets/css/map-controls.css" />
```
Comment (whamester) [inline pages/report-hazard/index.html:21]: import the css file inside the style.css

Comment (BryanCaldeira) [reply pages/report-hazard/index.html:21]: okay

**Review by ReinhardtBotha — APPROVED** [review]
looks good to me

**Comment by poojachauhan001** [comment]
image

 Need to add some padding below the zoom control. Also the bottom corners should be rounded.
 
 [Refer the UI here] - 
image

**Comment by poojachauhan001** [comment]
+ the text and address should be on one line.
image

image

**Review by whamester — APPROVED** [review]

**Pull request merged**